internal/provider/projects.go

```diff
@@ -0,0 +1,12 @@
+package provider
+
+type MeshProjectsSchema struct {
```

Comment:
f: This type belongs primarily to the client and should be called `MeshProjects`

Reply:
I've moved it to client as part of a nested response struct
https://github.com/meshcloud/terraform-provider-meshstack/blob/d77af63254ab0f1405fbd14537fe478f3146a31c/internal/provider/client.go#L307